src/components/TaskItem/TaskItem.tsx

```diff
@@ -13,10 +13,15 @@ const TaskItem = ({ title, checked, id, groupId }: TaskItemType) => {
   const [taskTitle, setTaskTitle] = useState(title);
   const [isTaskChecked, setIsTaskChecked] = useState(checked);
   const [isTaskEditable, setIsTaskEditbale] = useState(false);
+  const taskRef = useRef<HTMLDivElement>(null);
   const ref = useRef<HTMLInputElement>(null);
   useClickOutside(ref, () => setIsTaskEditbale(false), `ignoreClickOutside${id}`);
   const { updateTask, deleteTask } = useTaskStorage();
 
+  useEffect(() => {
+    taskRef.current?.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+  }, []);
```

Comment:
Same as the above but on the task item level - What would you like to achieve here by this useEffect?
Cause remember that fires on mount for every individual task

Reply:
Yes, on mount - just this first time, to scroll if view is in other area.